free-1.md

```diff
@@ -10,34 +10,67 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea helps people who are writing game rules to ensure that they are
+well-written without loopholes. Currently, game makers have to think hard about
+edge cases and not just try to figure out what they are, but how they should be
+handled so that they are most consistent with the rest of the rules. If they
+used a DSL stemming from my idea, they could be notified about what cases are
+left undecided, and the language could automatically help output their rules in
+clear English.
+
```

Comment:
Ooh. That is brilliant. Immediately I wonder whether such a thing could be only slightly extended to then also give you a game manager program. Although then, if it would not be much additional effort, I wonder if perhaps rigorously defining the logic of a game might end up being a significant challenge. In other words, it may have to be a general-purpose language, because of how variable games might be.

I'm immediately thinking of solutions to these problems. I guess that means this is an excellent idea.